modules/st2-auto-form/tests/test-array.js

```diff
@@ -30,11 +30,11 @@ describe('AutoForm ArrayField', () => {
 
     const c = new TestComponent(<ArrayField {...props} />);
 
-    c.makeChange('t,e,s,t');
+    c.makeChange('["t","e","s","t"]');
```

Comment:
No. It needs brackets now as indicated by the description. I could have handled those cases by first attempting to JSON parse the output then on failure attempting to split on `,` but this would be far less explicit and error prone. Additionally the UX for `object` requires braces to be included so this feels more consistent in my opinion.